README.md

```diff
@@ -1,66 +1,94 @@
 # Celestia Devnet
+- [Quickstart](#quickstart)
+- [Overview](#overview)
+- [Celestia Light Nodes](#celestia-light-nodes)
+- [Celestia Bridge Nodes](#celestia-bridge-nodes)
+- [Validator Bridge Nodes](#validator-bridge-nodes)
+- [Troubleshoot](#troubleshoot)
+
 > Note: This guideline is only relevant for the current devnet. As we approach to testnet, there will be a new guide
 
-- [Celestia Devnet](#celestia-devnet)
-  - [Celestia 101 (Overview)](#celestia-101-overview)
-    - [Celestia Full Node](#celestia-full-node)
-    - [Celestia Light Node](#celestia-light-node)
-  - [Installation guide](#installation-guide)
-  - [Troubleshoot](#troubleshoot)
+## Quickstart
+
+Devnet participants have the option of running:
+
+1. [Celestia Light Nodes](#celestia-light-nodes) (low CPU, 5GB+ disk): [get started](celestia-light-node.md)
+2. [Bridge Nodes](#bridge-nodes) (low CPU, 100GB+ disk): [get started](celestia-bridge-node.md)
+3. [Bridge Validator Nodes](#bridge-validator-nodes) (same node as 2): [get started](celestia-bridge-node.md#run-a-validator-bridge-node)
+4. _Celestia Full DA Nodes are under development_: [see ADR](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-003-march2022-testnet.md#full-node)
+
+You can view chain activity at the current `devnet-2` explorer: https://celestia.observer/
+## Overview
+
+Devnet demonstrates Celestia’s data availability capabilities on a libp2p network and live blockchain. 
+
+The current Devnet implementation runs two individual but connected networks:
+
+1. A libp2p network with [**Light Nodes**](#celestia-light-nodes), that handles data availability interactions
+2. A p2p network with [**Validators Nodes**](#bridge-validator-nodes), that handles the underlying consensus and block production. 
+
+Special [**Bridge Nodes**](#bridge-nodes) process blocks from the underlying consensus network to the data availability network.
```

Comment:
I propose to move this into the 1/1.1/1.2 points as the context for the reader could be grasped even more better 
wdyt?

Reply:
Sounds good! I rearranged the sections, and merged teh `components` explainers into this `overview` section